rpcs3/Emu/Io/usio.cpp

```diff
@@ -212,15 +212,15 @@ void usb_device_usio::translate_input_taiko()
 				{
 				case usio_btn::test:
 					if (player != 0) break;
-					if (pressed && !test_key_pressed) // Solve the need to hold the Test key
-						test_on = !test_on;
-					test_key_pressed = pressed;
+					if (pressed && !per_io_status[0].test_key_pressed) // Solve the need to hold the Test key
```

Comment:
Just do
`io_status& status = per_io_status[0];`
And use that below. Then you don't have to access the correct index in every following usage.

Same goes for the Tekken class.

Reply:
Done. Thanks!